### ISSUE TITLE
[css-inline-3] kerning of initial letter when initial letter has negative side bearing

### ISSUE THREAD

**Issue by dauwhe** [w3c/csswg-drafts#5476]
https://drafts.csswg.org/css-inline/#initial-letter-inline-position

> If the initial letter is a non-atomic inline with an automatic inline size and zero padding and borders, its margin box is kerned (negatively inset) by the distance from the start edge of its content box to the point in the content that would have been placed at the start edge of the line box if it were not an initial letter (i.e. the distance between its glyph bounding box and its start side bearing). This inset is effectively an additional inline-start margin on the box.

So the start side bearing can be either positive or negative. In Adobe Caslon Pro, the capital "V" has positive start side bearing—the left edge of the glyph bounding box is is at +22. So I presume we would move the initial-letter box left by that amount so the start edge of the glyph bounding box coincides with the start edge of the containing block. 

But the capital "Y" in the same font has negative start side bearing (-15). Would we then move the initial-letter box to the right? 

<img width="489" alt="Screen Shot 2020-08-26 at 3 54 46 PM" src="https://user-images.githubusercontent.com/5687700/91350308-7f0e6200-e7b4-11ea-9b47-4caeb2058ca2.png">

**Comment by fantasai** [comment]
Yes. Although seems what it probably *ought* to use is the [opbd feature](https://docs.microsoft.com/en-us/typography/opentype/spec/features_ko#tag-opbd) (if it exists) . The intention is to typeset the initial letter at the same position it would be if it were regular inline text. That's probably how we should describe it in the spec...

**Comment by fantasai** [comment]
Although I do wonder if that really makes the most sense. Maybe it should just be flush...

**Comment by dauwhe** [comment]
+1 to opdb. If that is absent, my instinct is that we should kern for positive side-bearing (as currently described in the spec) but leave negative side-bearing alone, for better optical alignment. But I am far from an expert here...
